spec/whois.md

```diff
@@ -0,0 +1,86 @@
+## The `whois` Folder
+
+The `did:webs` method defines that a controller MAY place a folder called
+`whois` within the root folder of the `did:webs` web location and place into
+that [signed files](#signed-files) that are [verifiable credentials] (VCs) or
```

Comment:
Does the fact that this mechanism references signed files mean that each VC or VP must have a .jws beside it, signed by the subject of the VC?

Reply:
Never mind; I see that you answer this later.